### PR TITLE
Update v3 build_fleet and build_manager

### PR DESCRIPTION
**Pull request by kyledong-suse**
1. Add clang17 and llvm17 into build_fleet for both amd64 and arm64
2. Use specific cs version 2.1.17 in build_manager for both amd64 and arm64

### CODE REVIEW

**Review by holyspectral — APPROVED** [review]
LGTM

**Review by gfsuse — APPROVED** [review]
LGTM

File: build/amd64/Dockerfile.build_manager_v3
[the remark this answers is not in the document]
Comment (jayhuang-suse) [reply build/amd64/Dockerfile.build_manager_v3:1]: Please ask UI team to provide a specific version if possible. I think we can use the current "hash" version temporally.

https://get-coursier.io/docs/2.1.16/cli-installation#specific-versions

**Review by jayhuang-suse — APPROVED** [review]
LGTM

**Review by jayhuang-suse — APPROVED** [review]

**Pull request merged**